### ISSUE TITLE
seattle_gov source defective

### ISSUE THREAD

**Issue by mampfes** [mampfes/hacs_waste_collection_schedule#99]
Source for seattle_gov not working any more:
 
```
 File "hacs_waste_collection_schedule/custom_components/waste_collection_schedule/waste_collection_schedule/source/seattle_gov.py", line 95, in fetch
    swServices = summary_info["accountSummaryType"]["swServices"][0]["services"]
KeyError: 'accountSummaryType'
```

Hi @gabosom, could you please help?

**Comment by atkulp** [comment]
Same here.  if i build up the URL and request it directly:

GET: `https://www.seattle.gov/UTIL/WARP/CollectionCalendar/GetCollectionDays?pApp=CC&pAddress=%27600%204th%20Ave%27&start=1636130000`

I get the following:

`[{"id":0,"title":null,"start":null,"end":null,"url":null,"allDay":false,"delimitedData":null,"status":"Cannot locate collection days.","Garbage":false,"Recycling":false,"FoodAndYardWaste":false}]`

It works if I don't quote the string though:

`https://www.seattle.gov/UTIL/WARP/CollectionCalendar/GetCollectionDays?pApp=CC&pAddress=600%204th%20Ave&start=1636130000`

I'll see if I can get a pull request in if you don't get a chance to look at it.

**Comment by atkulp** [comment]
I just noticed that the version installed to my HA instance is only 1.4.1.  For some reason it's not seeing newer versions...  I fixed mine manually for now, but not sure how to make the right version update.

**Comment by atkulp** [comment]
ok, weirder and weirder... the data coming back from the service is wrong.  City of Seattle has two different pages that will show you your schedule.  One is correct, the other is not.  Both should show recycling for today:

https://myutilities.seattle.gov/eportal/#/solidwasteservices

![image](https://user-images.githubusercontent.com/3195410/140550622-da25e7bd-c29e-436c-8a6b-577768ae825b.png)

https://www.seattle.gov/UTIL/WARP/CollectionCalendar/

![image](https://user-images.githubusercontent.com/3195410/140550643-8191a003-72dc-4635-bb24-14d49934e7ff.png)

I'm going to call them and see what's up.

**Comment by atkulp** [comment]
It appears that the UTIL/WARP URL is a forgotten relic not in use for many years.  It's amazing that it works at all (no one there seems to be aware of it anymore).  It looks like we need to get it working the prior way using the account and person ID's with the `https://myutilities.seattle.gov/rest/solidwastecalendar` url.

**Comment by atkulp** [comment]
Fixed in PR #100.  Awaiting review

**Comment by gabosom** [comment]
sorry, hadn't had any time to look into this. thx for fixing it. much appreciated
